openshift/console-plugin-template
# CONSOLE-3303: Move plugin template to the openshift GitHub org

### Screenshots: Running Cypress e2e tests in local env:
<img width="896" alt="Screenshot 2022-12-13 at 2 11 58 PM" src="https://user-images.githubusercontent.com/15249132/207460147-5c3833fe-73e4-4a44-bca6-9de0a354b412.png">

<img width="901" alt="Screenshot 2022-12-13 at 2 12 26 PM" src="https://user-images.githubusercontent.com/15249132/207460149-5b96bc44-55f5-4efb-8761-79f873c84506.png">

### Screencast:

https://user-images.githubusercontent.com/15249132/214329641-decd28ea-e644-4b62-a8ac-cb15d7d7a77d.mov

### Steps to test locally:
1. Build an image following the instructions in the main repo - [
console-plugin-template,](https://github.com/openshift/console-plugin-template#docker-image) and push to quay.io
2. Set the env variable:
`export CYPRESS_PLUGIN_TEMPLATE_PULL_SPEC=quay.io/<username>/console-plugin-template`
3. Run `yarn test-cypress`
Note: Helm is required to run the tests. See the instructions for installing helm [here](https://helm.sh/docs/intro/install/)
